spark/src/main/scala/geotrellis/spark/package.scala

```diff
@@ -127,6 +128,12 @@ package object spark
       }, preservesPartitioning = true)
   }
 
+  implicit class withRasterRDDSpatialKeyCropMethods(val self: RasterRDD[SpatialKey])
+      extends RasterRDDCropMethods[SpatialKey]
+
+  implicit class withRasterRDDSpaceTimeKeyCropMethods(val self: RasterRDD[SpaceTimeKey])
```

Comment:
these implicits should be defined in an `Implicits` trait in the package (`crop`). An object named Implicits in that package extends that (companion object), and then we extent the package with the Implicits trait